frontends/concrete-python/tests/execution/test_bitwise.py

```diff
@@ -59,7 +59,7 @@ def test_bitwise(function, parameters, helpers):
     circuit = compiler.compile(inputset, configuration)
 
     sample = helpers.generate_sample(parameters)
-    helpers.check_execution(circuit, function, sample)
+    helpers.check_execution(circuit, function, sample, retries=3)
```

Comment:
Can't we just take a better p_error ?

Reply:
Actually, even with a better p_error we need to retry it's still possible to have some failure even with a small p_error. By the way we use the p_error for those tests, maybe using the global one could be more appropriate as that the actual probability of fail should not depends of the test.
And as well that should come with better tests of the backends. So for we is good to merge as is

Reply:
Actually, we're using `global_p_error=(1 / 100_000)` so it's good!